core/src/main/java/com/sequenceiq/cloudbreak/service/cost/ClusterCostService.java

```diff
@@ -37,14 +40,20 @@ public class ClusterCostService {
     @Inject
     private EntitlementService entitlementService;
 
+    @Inject
+    private Map<CloudPlatform, PricingCache> pricingCacheMap;
+
     private boolean usdCalculationEnabled;
 
     public Map<String, RealTimeCost> getCosts(List<String> clusterCrns) {
         checkIfCostCalculationIsEnabled();
         errorIfCostCalculationFeatureIsNotEnabled();
         Map<String, RealTimeCost> realTimeCosts = new HashMap<>();
 
-        List<StackView> stacks = stackDtoService.findNotTerminatedByCrns(clusterCrns);
+        List<StackView> stacks = stackDtoService.findNotTerminatedByCrns(clusterCrns)
```

Comment:
I don't know how often this will be called, but maybe we should filter on the db side, so we won't bring all the eg azure stack from db to java side just to filter it out

Reply:
https://github.com/hortonworks/cloudbreak/pull/14060